Packs/ThreatGrid/Integrations/ThreatGridv2/ThreatGridv2.py

```diff
@@ -1006,8 +998,9 @@ def schedule_command(args: dict[str, Any], client: Client) -> PollResult:
     Returns:
         ScheduledCommand: Command, args, timeout and interval for CommandResults.
     """
-
+    first_run = False
```

Comment:
You can add the `hide_polling_output` argument in the yml file that does this.
Look into the `polling_function` decorator to see it, and you can see an example in `CrowdStrikeFalcon`.
More details here: https://xsoar.pan.dev/docs/integrations/scheduled-commands#the-pollresult-class:~:text=to%20Ignore%20Scheduled%20War%20Room%20Entries%20(as%20indicated%20below)%20add%20hide_polling_output%20as%20a%20boolean%20argument%20to%20the%20command%20in%20the%20yml%20file.